tools/deployment/osquery.initd

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+#
+# osqueryd 	Start/Stop the osquery daemon.
+#
+# chkconfig: 3345 90 60
+# Description: 
+#             With osquery, you can use SQL to query low-level
+#             operating system information. Under the hood, instead
+#             of querying static tables, these queries dynamically execute 
+#             high-performance native code. The results of the 
+#             SQL query are transparently returned to you quickly and easily 
+#              
+### BEGIN INIT INFO
+# Provides: osquery osqueryd 
+# Required-Start: $local_fs $syslog
+# Required-Stop: $local_fs $syslog
+# Default-Start:  3345
+# Default-Stop: 90
+# Short-Description: run osqueryd daemon
+# Description: 
+#             With osquery, you can use SQL to query low-level
+#             operating system information. Under the hood, instead
+#             of querying static tables, these queries dynamically execute 
+#             high-performance native code. The results of the 
+#             SQL query are transparently returned to you quickly and easily 
+#              
+#              
+### END INIT INFO
+
+
+RETVAL=0
+prog="osqueryd"
+exec=/usr/bin/osqueryd
+config=/etc/osquery/osquery.conf
+lockfile=/var/lock/subsys/osqueryd
+pidfile=/var/run/osquery.pid
+
+[ $UID -eq 0 ] && [ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
+
+. /etc/init.d/functions
+
+if [ ! -e $config ] ; then
+    echo "No osqueryd config file found! at $config"
```

Comment:
We're currently installing an example config at /var/osquery/osquery.example.conf. Maybe we should change this location to /usr/share/osquery/osquery.example.conf and echo a note here that points to this /usr/share/osquery location for examples?